### ISSUE TITLE
Display thread when click on message

### ISSUE THREAD

**Issue by oricou** [RocketChat/Rocket.Chat#20893]
### Description:

If the thread pannel is open (on the right) and I click on a message (in the middle) then nothing changes, I still see the thread of the previous message. To see the thread of a new message I have to click on "Reply" even if I don't want to reply.

### Steps to reproduce:

Click on a message with Thread panel opened

### Expected behavior:

If the thread pannel is open, display the thread of a message when we click on it.

### Actual behavior:

Nothing changes when we click on a message.

### Server Setup Information:

- Version of Rocket.Chat Server: 
- Operating System: Linux
- Deployment Method: I don't know (server of my school)
- Number of Running Instances: 
- DB Replicaset Oplog: 
- NodeJS Version: 

### Client Setup Information

- Desktop App or Browser Version: browser
- Operating System: linux

### Additional context

You issue form is a pain in the ass

### Relevant logs:

None

**Comment by yash-rajpal** [comment]
@oricou Please watch the language.
Also I am not able to reproduce the issue on latest develop branch. What is your server version?

You see the importance of this issue template, if you had filled necessary details I wouldn't be asking this question regarding server version.

Please confirm this from GIF below if this is what you are asking for?
![thread_query](https://user-images.githubusercontent.com/58601732/109023572-edd86380-76e2-11eb-809c-f4f63b56c237.gif)

**Comment by yash-rajpal** [comment]
@oricou Please watch the language.
Also I am not able to reproduce the issue on latest develop branch. What is your server version?

You see the importance of this issue template, if you had filled necessary details I wouldn't be asking this question regarding server version.

Please confirm this from GIF below if this is what you are asking for?
![thread_query1](https://user-images.githubusercontent.com/58601732/109024174-79ea8b00-76e3-11eb-8214-6c84eba52065.gif)

**Comment by oricou** [comment]
Yes, your behaviour is exactly what I expected. Since my sys adm told me to do a bug repport, I expected that what we had was the regular behaviour.
I will ask him for more information about the version he installed.

**Comment by oricou** [comment]
You click on the first reply. I want the same if I click on the message.

Note that sometimes the first reply is not visible.
